### ISSUE TITLE
Proptypes error with react 15.5.X and up

### ISSUE THREAD

**Issue by tiaaaa123** [artisavotins/react-redux-i18n#32]
```
 Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in  React v16.0. Use the latest available v15.* prop-types package from npm instead. For info on usage, compatibility, migration and more, see https://fb.me/prop-types-docs
```

**Comment by RodrigoCarvalho21** [comment]
Any updates on this?

**Comment by flo-sch** [comment]
Problem of `react-i18nify` package itself, solved by `1.9.2` release.

I guess this can be closed?
